### PR TITLE
Dragon Ball fusions

### PR DESCRIPTION
**Pull request by krikic**
Game of dragon Ball Fusion european region

### CODE REVIEW

**Comment by krikic** [comment]
Game Dragon Ball Fusions

**Comment by krikic** [comment]
Game platform fighter

File: Dragon Ball Fusions/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Dragon Ball Fusions"
+description = "The Dragon Ball Fusions is an fighter platform video game co-developed Nintendo 3DS handheld game console.
```
Comment (j-selby) [inline Dragon Ball Fusions/game.dat]: Check your grammar - potentially something like "Dragon Ball Fusions (ドラゴンボールフュージョンズ) is a role-playing video game for the Nintendo 3DS, based on the Dragon Ball franchise."
Missing a closing quote (") at the end.

File: Dragon Ball Fusions/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description = "The Dragon Ball Fusions is an fighter platform video game co-developed Nintendo 3DS handheld game console. 
+github_issues = [1654]
+needs_system_files = false
+needs_shared_font = false
+
+[[ releases ]]
+title = "0004000000125600"
+region = "EUR"
+release_date = "2017-02-17"
+
```
Comment (j-selby) [inline Dragon Ball Fusions/game.dat]: Could you drop this extra space? Thanks

**Comment by j-selby** [comment]
Superseded. Next time, don't create multiple PRs. thanks.

**Pull request closed without merging**